package.json

```diff
@@ -180,30 +174,30 @@
     ]
   },
   "scripts": {
+    "clean:build": "rimraf dist",
+    "clean:umd": "rimraf dist/umd",
     "prepublishOnly": "yarn install && yarn run build && yarn run test:noPrompt",
     "predeploy": "yarn run build-gh && cp styleguide/index.html styleguide/404.html && cp favicon.png styleguide/favicon.png",
     "build-gh": "yarn run build:docs",
     "deploy": "gh-pages -d styleguide && gh-pages -d styleguide -o upstream",
     "deploy:origin": "yarn install && yarn run predeploy && gh-pages -d styleguide && gh-pages -d styleguide -o origin",
-    "build-css": "sass src/lib/style/main.scss src/lib/style/main.css --load-path=. && sass src/demo/style:src/demo/style --load-path=.",
-    "watch-css": "yarn run build-css && sass src --watch -I .",
+    "watch:css": "yarn run build:css && sass src --watch --load-path=node_modules",
     "start-js": "react-scripts start",
-    "start": "npm-run-all -p watch-css serve:docs",
-    "start-deprecated-demo": "npm-run-all -p watch-css start-js",
+    "start": "npm-run-all -p watch:css serve:docs",
+    "start-deprecated-demo": "npm-run-all -p watch:css start-js",
     "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!(lodash-es|jest*)/)\" , --env=jsdom --setupFilesAfterEnv ./src/setupTests.js --resetMocks false",
     "test:coverage": "yarn test --coverage",
     "test:noPrompt": "CI=true yarn test",
     "eject": "react-scripts eject",
     "type-check": "tsc --noEmit",
     "type-check:watch": "yarn run type-check --watch",
-    "build": "yarn run build-css && yarn run build:js && yarn run build:copy-assets && yarn build:rollup",
+    "build": "yarn clean:build && yarn run build:css && yarn run build:js && yarn run build:copy-assets && yarn build:esbuild",
+    "build:css": "sass src/lib/style/main.scss src/lib/style/main.css --load-path=node_modules && sass src/demo/style:src/demo/style --load-path=node_modules",
```

Comment:
Dropping '.' from sass load-path (unnecessary) and adding `node_modules`, which is more common. IIRC the only reason we didn't do this before was because of a rollup plugin issue.